### PR TITLE
Contact form features

### PR DESCRIPTION
**Pull request by WilfriedM01**
1. Linters are setting up
2. README updated
3. Added the contact form that includes:
      - Name field accepts up to 30 chars.
      - Email field accepts only emails.
      - Message field accepts up to 500 chars.
      - All fields have placeholder text.
      - All fields have the correct type.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -328,7 +328,55 @@ <h3>CONOPY <span>&bull; Back End Dev &bull; 2015</span></h3>
...
+            <div class="form-container">
+              <h2 class="form-title">Contact me</h2>
+              <p class="form-description">
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding. I’d
+                love to help with it!
+              </p>
+              <form action="" method="post" class="contact-form">
+                <ul>
+                  <li class="name-field">
```
Comment (iambenkis) [inline index.html:343]: Kindly make sure the design of your form stick to the Figma design , It means the border , margin , size , font found in the Figma template should be represented in your design 😄 , Kindly improve the width of your `input`, etc

File: index.html [hunk 2]
```diff
@@ -328,7 +328,55 @@ <h3>CONOPY <span>&bull; Back End Dev &bull; 2015</span></h3>
...
+        <section class="form-wrap-level-one">
+          <div class="form-wrap-level-two">
+            <div class="form-container">
+              <h2 class="form-title">Contact me</h2>
+              <p class="form-description">
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding. I’d
+                love to help with it!
+              </p>
+              <form action="" method="post" class="contact-form">
```
Comment (iambenkis) [inline index.html:341]: Make sure that the `POST` method of your form is working properly, the user should be able to send a message using  the `action` and `method` attribute  integrated with [formspree service](https://formspree.io/html).

* `action` value should be similar to `https://formspree.io/f/{form_id}`
* `method` value equals `POST`Form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html).

Please, follow the link above and make sure the response is `successful`

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @WilfriedM01   
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by PinkMoon25 — APPROVED** [review]
Hi @amarmahdi & @WilfriedM01 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Pair Programming :heavy_check_mark:
Linters :heavy_check_mark: 
GitHub flow :heavy_check_mark: 
Changes Implemented :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**